announcements/_posts/2020-02-21-w06-linux-gaming.md

```diff
@@ -0,0 +1,29 @@
+---
+author: Jeffery Russell (@jrtechs)
+title: "Week #6: Linux Gaming"
+layout: post-event
+date-start: "2020-02-21 16:30"
+date-end: "2020-02-21 18:00"
+location: "GOL-2620 (Med. DB Lab)"
+---
+
+Hey RITluggers!
+
+This week Dev is giving a talk on the exciting developments in Linux gaming. 
+
+After years of anticipation, 
+Linux finally has some traction in the game dev industry. 
+Since 2013, things have only gotten exponentially better for gaming on Linux.
```

Comment:
What's with the indentation of splitting up the sentence? And lines 15/16 are kinda the same thing. Just worded a little odd.

Reply:
I'm gonna drop line 14